### DISCUSSION TITLE
How to view 4.0+ documentation

### DISCUSSION THREAD

**Discussion by FeiSuperMe** [dotnet/MQTTnet#2058] [Q&A] [answered]
The Client tab in this wiki page has the following description:
These samples are only valid for version 3. Please check the Samples directory for samples and documentation for version 4.0+.

Where can I get this 4.0+ documentation from?

**Comment by SeppPenner** [comment, marked as answer]
There is no wiki / documentation for that but examples are under https://github.com/dotnet/MQTTnet/tree/master/Samples.

**Reply by FeiSuperMe** [reply]
Ok.Thank you for your reply.

**Comment by OlegUA** [comment]
I'm trying to use MQTTnet.4.3.7.1207\lib\net461. But the MqttClientFactory from samples doesn't exist in this .dll

**Reply by chkr1011** [reply]
https://github.com/dotnet/MQTTnet/wiki There a link so the samples for version 4 and 5.

**Reply by OlegUA** [reply]
Thank you so much. Only one notice - the `using MQTTnet; ` must be added to all examples.
